Task_DEV_5/Task_DEV_5/FirstTextSorter.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.IO;
+
+namespace task_DEV_5
+{
+    class FirstTextSorter
+    {
+        public string inFile;
+        public string outFile;
+        public string[] contents;
+
+        public FirstTextSorter(string inFile, string outFile)
+        {
+            this.inFile = inFile;
+            this.outFile = outFile;
+        }
+
+        public void ReadText()
```

Comment:
Not sure that it is necessary to create such method. It looks like just wrapper around File.ReadAllLines.

Reply:
Done.